src/libstd/net/addr.rs

```diff
@@ -632,7 +746,9 @@ pub trait ToSocketAddrs {
     ///
     /// # Errors
     ///
-    /// Any errors encountered during resolution will be returned as an `Err`.
+    /// Any errors encountered during resolution will be returned as an [`Err`].
```

Comment:
This is obvious from the function signature with `Result` that it returns an `Err` on error… Or you would have added a "Panics" section instead 😉 Maybe instead mention which Err variants it will return? Apparently one/some of std::io::Error?